src/boss/cloudvm/worker_config_template.go

```diff
@@ -2,13 +2,14 @@ package cloudvm
 
 import (
 	"fmt"
+	"log"
 	"os"
 	"path/filepath"
 
 	"github.com/open-lambda/open-lambda/ol/common"
 )
 
-func LoadWorkerConfigTemplate(templatePath string, workerPath string) error {
+func LoadWorkerConfigTemplate(templatePath string, workerPath string, workerPort string) error {
 	// Create template.json if it doesn't exist
 	if _, err := os.Stat(templatePath); os.IsNotExist(err) {
```

Comment:
Instead of checking whether we need to initialize the template each time, why not do this in NewLocalWorkerPool?

Also, common.LoadConf loads the config to a global variable, which we then change, and without lock protection.  That's risky.  Can we make a version of common.LoadConf (w/ a different name) that returns a worker config instead of updating the global variable?   Then we could save the loaded config template in another field in LocalWorkerPoolPlatform, with full control over it and with locking (if needed).